packages/react-heartwood-components/src/components/Forms/components/Autosuggest/Autosuggest.js

```diff
@@ -1,11 +1,15 @@
 // @flow
 import React, { Component } from 'react'
+import { createPortal } from 'react-dom'
+import debounce from 'lodash/debounce'
 import { default as ReactAutosuggest } from 'react-autosuggest'
 import cx from 'classnames'
 import Button from '../../../Button/Button'
 import { InputPre, InputHelper } from '../../FormPartials'
 import ClearIcon from '../../../../../static/assets/icons/ic_cancel.svg'
 
+// TODO: Figure out max height setting. Should be based on results length or manual? Both?
```

Comment:
@jtini will default to you on this but will probably want to implement before merging so we don't add additional tech debt to the implementation in skills/web. My thought is that it could have a default max height (as it does now) but the option to set the height based on the number of preferred visible rows?